proposals/lip-0046.md

```diff
@@ -137,38 +145,36 @@ The function returns a 16-bytes value, `digest`.
 ##### Execution
```

Comment:
You could also remove the Parameters and Returns section, it is clear from the function description and type hinting.

Reply:
Done. Added a one-sentence explanation on the "hashing function" subsection.